.github/workflows/check-new-versions.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository code
-        uses: actions/checkout@1d96c772d19495a3b5c517cd2bc0cb401ea0529f # v4.1.3
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
Whats the reason for downgrading the the version here?

Reply:
ohh great catch - p sure that came from https://github.com/rancher/kubectl/pull/131 being merged a sloppy rebase.
will fix RN

Reply:
rebased to amend commit to not downgrade. 😄